tests/unit_tests/test_wallet.py

```diff
@@ -15,13 +15,200 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+import json
 import time
 import pytest
+import random
+import getpass
 import unittest
 import bittensor
+from rich.prompt import Confirm
+from ansible_vault import Vault
 from unittest.mock import patch, MagicMock
 
 
+class TestWalletUpdate(unittest.TestCase):
+    def setUp(self):
+        self.default_updated_password = "nacl_password"
+        self.default_legacy_password = "ansible_password"
+        self.empty_wallet = bittensor.wallet(name=f"mock-empty-{str(time.time())}")
+        self.legacy_wallet = self.create_legacy_wallet() 
+        self.wallet = self.create_wallet()
+
+    def legacy_encrypt_keyfile_data(keyfile_data: bytes, password: str = None) -> bytes:
+        console = bittensor.__console__
+        with console.status(":locked_with_key: Encrypting key..."):
+            vault = Vault(password)
+        return vault.vault.encrypt(keyfile_data)
+    
+    def create_wallet(self):      
+        # create an nacl wallet
+        wallet = bittensor.wallet(name=f"mock-{str(time.time())}")  
+        with patch.object(
+            bittensor,
+            "ask_password_to_encrypt",
+            return_value=self.default_updated_password,
+        ):
+            wallet.create()
```

Comment:
By default, all wallets created from now on will be NaCl encrypted right?

Reply:
yes yes